.github/workflows/go.yml

```diff
@@ -49,22 +49,20 @@ jobs:
     - name: Vet
       if: matrix.os == 'ubuntu-latest'
       run: make vet
-    - name: Test with Race Detector
-      # do not tests with race detector on windows
-      # until the issue with badgerdb memory allocation
-      # is solved in this case
-      if: matrix.os != 'windows-latest'
+    - name: Test with race detector & code coverage
+      if: matrix.os == 'ubuntu-latest'
+      # only generate code coverage profile on ubuntu
       run: make cover=1 test-race
-    - name: Test
-      if: matrix.os == 'windows-latest'
-      run: make test
-    - name: Coveralls
+    - name: Test with race detector
+      if: matrix.os != 'ubuntu-latest'
+      run: make test-race
```

Comment:
@acud you will also execute race-tests on Windows, I know that they were disabled for Windows, maybe its time to enable them again, don't know

Reply:
yep i enabled them on purpose :+1: